### PR TITLE
Updated the content and the links due to mono repo changes.

### PR DESCRIPTION
**Pull request by Rashmirreddy**

### CODE REVIEW

File: content/md/en/docs/reference/how-to-guides/pallet-design/use-loose-coupling.md [hunk 1]
```diff
@@ -151,9 +151,9 @@ To update the runtime configuration for your pallet:
 
 ## Examples
 
-- [`EnsureOrigin`](https://paritytech.github.io/substrate/master/frame_support/traits/trait.EnsureOrigin.html) trait in the [Democracy pallet](https://github.com/paritytech/substrate/blob/master/frame/democracy/src/lib.rs#L294-L352)
-- [Weighting methods](https://github.com/paritytech/substrate/blob/master/frame/identity/src/weights.rs#L46-L64) in the [Identity pallet](https://github.com/paritytech/substrate/blob/master/frame/identity/src/lib.rs#L149-L151).  
--[`KeyOwnerProofSystem`](https://paritytech.github.io/substrate/master/frame_support/traits/trait.KeyOwnerProofSystem.html) in [Grandpa pallet](https://github.com/paritytech/substrate/blob/master/frame/grandpa/src/lib.rs#L106).
+- [`EnsureOrigin`](https://paritytech.github.io/substrate/master/frame_support/traits/trait.EnsureOrigin.html) trait in the [Democracy pallet](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/democracy/src/lib.rs#L294-L352)
```
Comment (wentelteefje) [inline content/md/en/docs/reference/how-to-guides/pallet-design/use-loose-coupling.md:154]:
```suggestion
- [`EnsureOrigin`](https://paritytech.github.io/substrate/master/frame_support/traits/trait.EnsureOrigin.html) trait in the [Democracy pallet](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/democracy/src/lib.rs#L298-L335)
```

File: content/md/en/docs/reference/how-to-guides/pallet-design/use-loose-coupling.md [hunk 2]
```diff
@@ -151,9 +151,9 @@ To update the runtime configuration for your pallet:
...
+- [Weighting methods](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/identity/src/weights.rs#L46-L64) in the [Identity pallet](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/identity/src/lib.rs#L149-L151).  
+-[`KeyOwnerProofSystem`](https://paritytech.github.io/substrate/master/frame_support/traits/trait.KeyOwnerProofSystem.html) in [Grandpa pallet](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/grandpa/src/lib.rs#L106).
```
Comment (wentelteefje) [inline content/md/en/docs/reference/how-to-guides/pallet-design/use-loose-coupling.md:156]:
```suggestion
-[`KeyOwnerProofSystem`](https://paritytech.github.io/substrate/master/frame_support/traits/trait.KeyOwnerProofSystem.html) in [Grandpa pallet](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/grandpa/src/lib.rs#L111-L122).
```

File: content/md/en/docs/reference/how-to-guides/parachains/connect-to-a-relay-chain.md [hunk 3]
```diff
@@ -180,10 +180,10 @@ pub const fn deposit(items: u32, bytes: u32) -> Balance {}
 You can find this function for each relay chain in the Polkadot repository.
 For example:
 
-- [Kusama](https://github.com/paritytech/polkadot/blob/master/runtime/kusama/constants/src/lib.rs)
-- [Polkadot](https://github.com/paritytech/polkadot/blob/master/runtime/polkadot/constants/src/lib.rs)
-- [Rococo](https://github.com/paritytech/polkadot/blob/master/runtime/rococo/constants/src/lib.rs)
-- [Westend](https://github.com/paritytech/polkadot/blob/master/runtime/westend/constants/src/lib.rs)
+- [Kusama](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/runtime/kusama/constants/src/lib.rs)
+- [Polkadot](https://github.com/paritytech/polkadot/blob/master/rhuntime/polkadot/constants/src/lib.rs)
```
Comment (wentelteefje) [inline content/md/en/docs/reference/how-to-guides/parachains/connect-to-a-relay-chain.md:184]:
```suggestion
- [Polkadot](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/runtime/polkadot/constants/src/lib.rs)
```

File: content/md/en/docs/reference/how-to-guides/tools/use-try-runtime.md [hunk 4]
```diff
@@ -27,7 +27,7 @@ As with pallets, be sure that you are using the appropriate tag or branch for th
   
    ```toml
    [dependencies]
-	 frame-try-runtime = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v1.0.0", optional = true }
+	 frame-try-runtime = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "polkadot-v1.0.0", optional = true }
```
Comment (wentelteefje) [inline content/md/en/docs/reference/how-to-guides/tools/use-try-runtime.md:30]: Same comment about the `branch` as above

File: content/md/en/docs/reference/how-to-guides/weights/add-benchmarks.md [hunk 5]
```diff
@@ -7,19 +7,19 @@ keywords:
...
-1. Open the [`Cargo.toml`](https://github.com/paritytech/substrate/blob/master/frame/examples/basic/Cargo.toml) file for your pallet in a text editor.
+1. Open the [`Cargo.toml`](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/examples/basic/Cargo.toml) file for your pallet in a text editor.
    
 2. Add the `frame-benchmarking` crate to the [dependencies] for the pallet using the same version and branch as the other dependencies in the pallet.
    
    For example:
 
    ```toml
-   frame-benchmarking = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v1.0.0", optional = true }
+   frame-benchmarking = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/polkadot-sdk.git", branch = "polkadot-v1.0.0", optional = true }
```
Comment (wentelteefje) [inline content/md/en/docs/reference/how-to-guides/weights/add-benchmarks.md:22]: again the `branch`

File: content/md/en/docs/reference/how-to-guides/weights/use-conditional-weights.md [hunk 6]
```diff
@@ -115,7 +115,7 @@ Use the conditional weighting struct on your pallet's extrinsics like this:
 
 ## Examples
 
-- [Custom `WeightForSetDummy` struct in the Example pallet](https://github.com/paritytech/substrate/blob/master/frame/examples/basic/src/lib.rs)
+- [Custom `WeightForSetDummy` struct in the Example pallet](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/examples/basic/src/lib.rs)
```
Comment (wentelteefje) [inline content/md/en/docs/reference/how-to-guides/weights/use-conditional-weights.md:118]: Proper lines highlighted 
```suggestion
- [Custom `WeightForSetDummy` struct in the Example pallet](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/examples/basic/src/lib.rs#L305-L350)
```

File: content/md/en/docs/reference/xcm-reference.md [hunk 7]
```diff
@@ -446,7 +446,7 @@ Transact {
 },
 ```
 
-For additional examples of converting origins, see the [prigin_conversion](https://github.com/paritytech/polkadot/blob/master/xcm/xcm-builder/src/origin_conversion.rs) module.
+For additional examples of converting origins, see the [prigin_conversion](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/xcm/xcm-builder/src/origin_conversion.rs) module.
```
Comment (wentelteefje) [inline content/md/en/docs/reference/xcm-reference.md:449]:
```suggestion
For additional examples of converting origins, see the [origin_conversion](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/xcm/xcm-builder/src/origin_conversion.rs) module.
```

File: content/md/en/docs/test/set-up-a-test-network.md [hunk 8]
```diff
@@ -47,7 +47,7 @@ To prepare a working folder with the binaries for the test network:
 1. Clone the Polkadot repository by running a command similar to the following:
 
    ```bash
-   git clone --depth 1 --branch release-v1.0.0 https://github.com/paritytech/polkadot.git
+   git clone --depth 1 --branch release-v1.0.0 https://github.com/paritytech/polkadot-sdk.git
    ```
 
    Release branches use the naming convention `release-v<n.n.n>`.
```
Comment (wentelteefje) [inline content/md/en/docs/test/set-up-a-test-network.md:53]: As of now, these don't exist in `polkadot-sdk`.

File: content/md/en/docs/tutorials/build-a-blockchain/authorize-specific-nodes.md [hunk 9]
```diff
@@ -131,7 +131,7 @@ To add the `node-authorization` pallet to the Substrate runtime:
 
    ```toml
    [dependencies]
-   pallet-node-authorization = { default-features = false, version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v1.0.0" }
+   pallet-node-authorization = { default-features = false, version = "4.0.0-dev", git = "https://github.com/paritytech/polkadot-sdk.git", branch = "polkadot-v1.0.0" }
```
Comment (wentelteefje) [inline content/md/en/docs/tutorials/build-a-blockchain/authorize-specific-nodes.md:134]: Again `branch`

File: content/md/en/docs/tutorials/build-a-blockchain/upgrade-a-running-network.md [hunk 10]
```diff
@@ -149,7 +149,7 @@ To update the dependencies for the runtime to include the Utility pallet:
    codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = ["derive"] }
    scale-info = { version = "2.1.1", default-features = false, features = ["derive"] }
    
-   pallet-aura = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v1.0.0" }
+   pallet-aura = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/polkadot-sdk.git", branch = "polkadot-v1.0.0" }
```
Comment (wentelteefje) [inline content/md/en/docs/tutorials/build-a-blockchain/upgrade-a-running-network.md:152]: :(

File: content/md/en/docs/tutorials/build-a-parachain/prepare-a-local-relay-chain.md [hunk 11]
```diff
@@ -62,7 +62,7 @@ Therefore, this tutorial uses code from the Polkadot repository to prepare the l
    You can find information about each release on the [Releases](https://github.com/paritytech/polkadot/releases) tab in GitHub.
    
    ```bash
-   git clone --branch release-v1.0.0 https://github.com/paritytech/polkadot.git
+   git clone --branch release-v1.0.0 https://github.com/paritytech/polkadot-sdk.git
```
Comment (wentelteefje) [inline content/md/en/docs/tutorials/build-a-parachain/prepare-a-local-relay-chain.md:65]: `branch`

File: content/md/en/docs/quick-start/explore-the-code.md [hunk 12]
```diff
@@ -88,11 +88,11 @@ Before you start to customize the runtime for your own project, you should spend
 You've already seen how the default manifest for the runtime lists the default dependencies and features for the runtime in lines similar to the following:
 
 ```rust
-pallet-balances = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-vX.Y.Z" }
+pallet-balances = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/polkadot-sdk.git", branch = "polkadot-vX.Y.Z" }
```
[the remark this answers is not in the document]
Comment (Rashmirreddy) [reply content/md/en/docs/quick-start/explore-the-code.md:91]: The new repo should be available in 2 weeks, clarified this with Sacha. Will change it after the repo is available, and till then we can go ahead with this.

File: content/md/en/docs/test/set-up-a-test-network.md (hunk 8 above)
Comment (Rashmirreddy) [reply content/md/en/docs/test/set-up-a-test-network.md:53]: The repo will be available soon, will modify it when it becomes available.

File: content/md/en/docs/tutorials/build-a-blockchain/authorize-specific-nodes.md (hunk 9 above)
Comment (Rashmirreddy) [reply content/md/en/docs/tutorials/build-a-blockchain/authorize-specific-nodes.md:134]: The new repo should be available in 2 weeks, clarified this with Sacha. Will change it after the repo is available, and till then we can go ahead with this.

File: content/md/en/docs/tutorials/build-a-blockchain/upgrade-a-running-network.md (hunk 10 above)
Comment (Rashmirreddy) [reply content/md/en/docs/tutorials/build-a-blockchain/upgrade-a-running-network.md:152]: The new repo should be available in 2 weeks, clarified this with Sacha. Will change it after the repo is available, and till then we can go ahead with this.

**Pull request merged**